docs/node/run-your-node/paratime-node.mdx

```diff
@@ -330,6 +296,136 @@ accessible on the internet (for `TCP` and `UDP` traffic).
 
 :::
 
+#### Set Custom Registry
+
+If you want to download bundles from a registry other than the
+[Oasis Bundle Registry], add the URL of your desired repository to the runtime
+configuration.
+
+```
+# ... sections not relevant are omitted ...
+runtime:
+    runtimes:
+        - id: {{ runtime_identifier }}
+            components: {}
+            config: {}
+    # Custom repository configuration
+    repositories:
+        - {{ url_to_repository }}
+```
+
+Before using this configuration you should collect the following information
+to replace the variables present in the configuration file:
+
+* `{{ runtime_identifier }}`: You can find the runtime identifier in the Network
+Parameters page ([Mainnet], [Testnet]).
+* `{{ url_to_repository }}`: Url to your custom repository.
+
+### Manual ParaTime Bundle Installation
+
+For the Emerald or custom ParaTime, configuring paths to ParaTime bundles is
+required. However, for the Sapphire and Cipher ParaTimes, this setup is
+optional, as bundles will be automatically downloaded by the Oasis node from the
+[Oasis Bundle Registry].
+
+ROFL bundles are managed either manually or automatically by the ROFL scheduler.
```

Comment:
Why are we mentioning this under ParaTimes. Maybe we create a new subsection for ROFL.